Android/AndroidSwipeLayout/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@mattleibow you now have 2 license files. Remove here and one on components folder.  Remove the components folder one and fix the yaml to point here.

Reply:
Done.